onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -1043,4 +1057,10 @@
     <string name="transit_mode_bus">Bus only</string>
     <string name="transit_mode_rail">Rail only</string>
 
+    <!-- Social -->
+    <string name="es_google_auth_callback">org.onebusaway.android</string>
```

Comment:
Can you please move `es_google_auth_callback` and `es_google_auth_redirect` to `donottranslate.xml`?  These strings aren't intended to be user-facing and `strings.xml` should only hold strings that are translatable.

Thinking more - should this actually be defined in `build.gradle` as part of the flavor?  Looks like it would be specific per white-labeled version of the app.